plugins/taotooltip/plugin.js

```diff
@@ -209,7 +209,7 @@ CKEDITOR.plugins.add('taotooltip', {
 		});
 
 		editor.ui.addButton('TaoTooltip', {
-			label: 'Tooltip',
+			label: editor.lang[commandName].button,
```

Comment:
maybe should be smth like `editor.lang.insertTaoTooltip.button`?

Reply:
It's for reuse the `commandName` variable create above and used in other places

Reply:
ok